src/docs/feature-launch-process.md

```diff
@@ -2,21 +2,41 @@
 title: 'Implementing and shipping JavaScript/WebAssembly language features'
 description: 'This document explains the process for implementing and shipping JavaScript or WebAssembly language features in V8.'
 ---
-In general, V8 uses the [Blink Intent process](https://www.chromium.org/blink/launching-features) for JavaScript and WebAssembly language features. The differences are laid out in the errata below. Please follow the Blink Intent process, unless the errata tells you otherwise.
+In general, V8 follows the [Blink Intent process for already-defined consensus-based standards](https://www.chromium.org/blink/launching-features/#process-existing-standard) for JavaScript and WebAssembly language features. V8-specific errata are laid out below. Please follow the Blink Intent process, unless the errata tells you otherwise.
 
-If you have any questions on this topic, please send hablich@chromium.org and v8-dev@googlegroups.com an email.
+If you have any questions on this topic for JavaScript features, please email syg@chromium.org and v8-dev@googlegroups.com.
+
+For WebAssembly features, please email gdeepti@chromium.org and v8-dev@googlegroups.com.
 
 ## Errata
 
+### JavaScript features usually wait until Stage 3+ { #stage3plus }
+
+As a rule of thumb, V8 waits to implement JavaScript feature proposals until they advance to [Stage 3 or later in TC39](https://tc39.es/process-document/). TC39 has its own consensus process, and Stage 3 or later signals explicit consensus among TC39 delegates, including all browser vendors, that a feature proposal is ready to implement. This external consensus process means Stage 3+ features do not need to send Intent emails other than Intent to Ship.
+
 ### TAG review { #tag }
 
 For smaller JavaScript or WebAssembly features, a TAG review is not required, as TC39 and the Wasm CG already provide significant technical oversight. If the feature is large or cross-cutting (e.g., requires changes to other Web Platform APIs or modifications to Chromium), TAG review is recommended.
 
+### Both V8 and blink flags are required { #flags }
+
+When implementing a feature, both a V8 flag and a blink `base::Feature` are required.
+
+Blink features are required so that Chrome can turn off features without distributing new binaries in emergency situations. This is usually implemented in [`gin/gin_features.h`](https://source.chromium.org/chromium/chromium/src/+/main:gin/gin_features.h), [`gin/gin_features.cc`](https://source.chromium.org/chromium/chromium/src/+/main:gin/gin_features.cc), and [`gin/v8_initializer.cc`](https://source.chromium.org/chromium/chromium/src/+/main:gin/v8_initializer.cc),
+
+### Fuzzing is required to ship { #fuzzing }
+
+JavaScript and WebAssembly features must be fuzzed, and all fuzz bugs must be fixed, before they can be shipped.
```

Comment:
Wdyt of being explicit about a feature being fuzzed for a minimum length of time for fuzzer coverage (4 weeks/1 milestone)?

Reply:
Yeah, sounds good to me.